spice/client-tests/zoom.py

```diff
@@ -38,6 +38,9 @@
                    const="mouse", dest="method")
 group.add_argument("-w", "--windowmanager", help="Use window manager.",
                    action="store_const", const="wm_key", dest="method")
+parser.add_argument("-r", "--repeat", help="Repeat zooming.", default=0, 
+                   dest="repeat", type=int)
+parser.add_argument("-R", "--reset", help="Reset to normal size.", action="store_true")
```

Comment:
Maybe it would be good to specify a moment of reset in help="Reset to normal size before exit."

...